modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -50,7 +50,7 @@ def execute cmd, msg=nil, error_only=false
       rescue Timeout::Error
         raise Proxy::Dns::Error.new("dnscmd did not respond within #{tsecs} seconds")
       ensure
-        std_in.close  unless std_in.nil?
+        std_in&.close
         std_out.close unless std_in.nil?
         std_err.close unless std_in.nil?
```

Comment:
While I have no idea about this code, I guess these two should get adjusted.

Reply:
This looks like a bug indeed

Reply:
Yeah definitely, just adjust everything to `&.` please.

Reply:
Note that it's always checking `std_in` here so the cop can't detect it's the same.